lib/line-protocol.js

```diff
@@ -36,11 +36,12 @@ internals.String = (value) => {
     return `"${string}"`;
 };
 
-internals.tags = (event) => {
+internals.tags = (event, config) => {
 
     return {
         host : event.host || internals.host,
-        pid  : event.pid
+        pid  : event.pid,
+        serviceName: config.metadata.serviceName
```

Comment:
If no metadata.serviceName is set then the line will be formatted as:

```
ops,host=mytesthost,pid=9876,serviceName=undefined
```

It would be cleaner to only add this property if  `config.metadata.serviceName` is set.